test/anvil/anvil-global-setup.ts

```diff
@@ -52,7 +52,7 @@ export const ANVIL_NETWORKS: Record<NetworksWithFork, NetworkSetup> = {
         rpcEnv: 'SEPOLIA_RPC_URL',
         fallBackRpc: 'https://sepolia.gateway.tenderly.co',
         port: ANVIL_PORTS.SEPOLIA,
-        forkBlockNumber: 6000510n,
+        forkBlockNumber: 6178545n,
```

Comment:
Set to a few blocks after the `MOCK_WETH_BAL_POOL` deployment as suggested [here](https://github.com/balancer/b-sdk/issues/334#issuecomment-2188311505)